### ISSUE TITLE
partial bug with ${message.from}

### ISSUE THREAD

**Issue by Yann-R** [robertklep/quotefixformac#82]
Hello,
When trying to customize the attribution, I observed something like a bug?

`${message.from}` is documented as `Name and e-mail address of sender of message` but it writes only the display name for my tests, then behaving like the other variable: `${message.from.name`

The workaround is then to use `${message.from/name}` combined with `${message.from.email}` to get the same as `${message.from}`

Normal?

**Comment by robertklep** [comment]
No, not normal.

It works for me, but I wouldn't be surprised if it doesn't work for some edge cases. It doesn't work at all, or just in particular cases?

**Comment by Yann-R** [comment]
Yes, always: I never get something else than the Display Name, without e-mail address.

As an example, with a simple template like `${message.from} wrote:`, the github notification I received from your above answer gives simply the following when clicking to reply:
`Robert Klep wrote:`

My system:
- macOS version 10.12.6
- Mail version 10.3 (3273), (even launched in English this doesn't solve the problem)
- QuoteFix version 2.8.0
- no options enabled in `Quotefixing tab` (equivalent to uncheck `Quotefixing is enabled`)

**Comment by Yann-R** [comment]
BTW, my observed bug is the same with QuoteFix version 2.8.2 that I just installed
(strangly, the embedded version checking did not tell me there was a newer version).
